app/scripts/controllers/projectlist.js

```diff
@@ -16,6 +16,7 @@ angular.module('mockuperApp')
             loginService.reloadScope();
             headerService.updateHeader('projects');
             $scope.userName = $rootScope.userNameLogin;
+
             io.socket.get('/project', function serverResponded(body, JWR) {
                 //console.log('project get');
```

Comment:
this line "//console.log('project get');" should be removed, I think comments are good only to describe certain complex code.